prime-router/src/main/kotlin/azure/DatabaseAccess.kt

```diff
@@ -1352,6 +1355,106 @@ class DatabaseAccess(private val create: DSLContext) : Logging {
             .fetchInto(Action::class.java)
     }
 
+    /**
+     * Returns all permissions associated with an organization by name
+     * @param organizationName an exact organization name
+     * @param txn an optional database transaction
```

Comment:
This question is just for my own edification, but do we need to start a transaction for single statements like one `SELECT`?

Reply:
Oh, duh -- it's not executed immediately at this point.  I should have just read a tiny bit further before typing that comment!  😅